code/datums/mutations/body.dm

```diff
@@ -375,3 +375,69 @@
 			owner.SetStun(owner.AmountStun()*2)
 			owner.visible_message("<span class='danger'>[owner] tries to stand up, but trips!</span>", "<span class='userdanger'>You trip over your own feet!</span>")
 			stun_cooldown = world.time + 300
+
+/datum/mutation/human/strongwings
+	name = "Strengthened Wings"
+	desc = "Subject's wing muscle volume rapidly increases."
+	quality = POSITIVE
+	difficulty = 12
+	instability = 15
+	species_allowed = list(SPECIES_APID, SPECIES_MOTH)
+
+/datum/mutation/human/strongwings/on_acquiring()
+	if(..())
+		return
+	if(locate(/obj/item/organ/wings/bee) in owner.internal_organs)
```

Comment:
`locate()` returns null if it doesn't find it. Only call `locate()` once and check for null for each of these.